src/GraphQl/Mutations/mutations.ts

```diff
@@ -84,21 +84,13 @@ export const ADDRESS_DETAILS_FRAGMENT = gql`
 
 export const UPDATE_USER_MUTATION = gql`
   mutation UpdateUserProfile(
-    $id: ID
     $firstName: String
     $lastName: String
     $email: EmailAddress
-    $applangcode: String
     $file: String
   ) {
     updateUserProfile(
-      data: {
-        firstName: $firstName
-        lastName: $lastName
-        email: $email
-        id: $id
-        applangcode: $applangcode
-      }
+      data: { firstName: $firstName, lastName: $lastName, email: $email }
```

Comment:
why did you removed applangcode, id from here ?

Reply:
Hi @SiddheshKukade, If I don't remove those fields from the schema, introspection test failed. Which is why I was told to remove them for now until [#1762](https://github.com/PalisadoesFoundation/talawa-api/pull/1762) merged in talawa-api side. The PRs of talawa-admin cannot be merged without removing those fields until the talawa-api PR is merged.